src/app/Fake.Core.Process/Process.fs

```diff
@@ -361,7 +361,7 @@ let tryFindFile dirs file =
                    |> String.replace "[ProgramFiles]" Environment.ProgramFiles
                    |> String.replace "[ProgramFilesX86]" Environment.ProgramFilesX86
                    |> String.replace "[SystemRoot]" Environment.SystemRoot
-                   |> DirectoryInfo.ofPath
```

Comment:
I'm really back and forth about this. We have `ofSeq` `ofList` and a lot of these everywhere. But you are right in that this meets the F# Guidelines better :/

Reply:
As said on Gitter, I asked people at FableConf, people seems to prefer camelCase for functions.